### PR TITLE
[Fix] Orders not creating from the POS for new customer if customer name is based on naming series

### LINKED ISSUE #8621: Amount field in POS should be read only
Hi,

The Amount field in the V8 POS should be read only. Some users inadvertently put the price of an item in the amount field and then proceed to payment screen and wonder why the figure isn't showing automatically in the mode of payment. Some even fill the mode of payment manually and submit the invoice! This leads to zero value invoices with status showing as 'Overdue'

Kind regards,

### LINKED ISSUE #8923: POS - When syncing offline invoices, getting error message
Steps followed for creating an invoice.

1. Created new Customer from the POS View itself.
1. Select items.
1. Created Payment Entry.
1. Menu > Sync Offline Invoices

On the last action, received following error message.

screen shot 2017-05-19 at 10 54 01 am

### LINKED ISSUE #9110: Duplicate Customer Error blocking queue in POS
Hi,

There's a Duplicate Customer Error blocking the queue in POS for the past 2 days. There's nothing in the error logs. Please find screenshot below:

pos duplicate customer issue 1

Please help fix asap

Thanks

### PR DESCRIPTION
**Pull request by rohitwaghchaure**
**Issue**
Customer name by naming series
screen shot 2017-05-19 at 2 38 07 pm

bbb6e51a-3c81-11e7-94cd-c80041a2e5ea

Fixed #8923 
Fixed #8621 
Fixed #9110

### CODE REVIEW

**Comment by nabinhait** [comment]
@mbauskar Can you test this?

**Review by mbauskar — APPROVED** [review]

**Review by mbauskar — CHANGES_REQUESTED** [review]
while syncing Sales Invoice the same customer is synced multiple times.

**Comment by rohitwaghchaure** [comment]
Fixed!

**Review by mbauskar — APPROVED** [review]

File: erpnext/selling/doctype/customer/customer.json [hunk 1]
```diff
@@ -1057,21 +1093,51 @@
...
+   "hidden": 0, 
+   "ignore_user_permissions": 0, 
+   "ignore_xss_filter": 0, 
+   "in_filter": 0, 
+   "in_global_search": 0, 
+   "in_list_view": 0, 
+   "in_standard_filter": 0, 
+   "label": "Customer POS ID", 
+   "length": 0, 
+   "no_copy": 0,
```
Comment (nabinhait) [inline erpnext/selling/doctype/customer/customer.json]: no_copy=1

File: erpnext/selling/doctype/customer/customer.json [hunk 2]
```diff
@@ -1057,21 +1093,51 @@
...
+   "in_filter": 0, 
+   "in_global_search": 0, 
+   "in_list_view": 0, 
+   "in_standard_filter": 0, 
+   "label": "Customer POS ID", 
+   "length": 0, 
+   "no_copy": 0, 
+   "permlevel": 0, 
+   "precision": "", 
+   "print_hide": 0,
```
Comment (nabinhait) [inline erpnext/selling/doctype/customer/customer.json]: print_hide=1

File: erpnext/selling/doctype/customer/customer.json [hunk 3]
```diff
@@ -1057,21 +1093,51 @@
...
+  }, 
+  {
+   "allow_bulk_edit": 0, 
+   "allow_on_submit": 0, 
+   "bold": 0, 
+   "collapsible": 0, 
+   "columns": 0, 
+   "fieldname": "customer_pos_id", 
+   "fieldtype": "Data", 
+   "hidden": 0,
```
Comment (nabinhait) [inline erpnext/selling/doctype/customer/customer.json]: hidden=1

**Review by nabinhait — CHANGES_REQUESTED** [review]

**Review by mbauskar — APPROVED** [review]

**Pull request merged**